### PR TITLE
Update Create Project Doc

### LINKED ISSUE #451: With astrocloud CLI, users no longer need to be in the directory they installed the CLI
**Affected Document:**
Step 1 instructions on creating an astro project: https://docs.astronomer.io/astro/create-project#step-1-create-an-astro-project
**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [X] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

This text should be updated: 

```
To create a new Astro project folder, open the directory where you installed the CLI and run: Astro
```

**Additional Notes:**
With the new astrocloud CLI, the commands are available regardless of which directory the user is in. The current instruction mentions "open the directory where you installed the CLI" which is no longer accurate. It also has a trailing `Astro` in the text.

Instead, I think the instructions should mention something like:
"Create a new Astro project folder and navigate to it, then initialize the project:"

```
mkdir my_astro_project
cd my_astro_project
astrocloud dev init
```

### PR DESCRIPTION
**Pull request by paolaperaza**
This should resolve #451 by:

- Updating language on the steps to create a project
- Adds our "Develop Project" doc to next steps for increased cross-linkage

### CODE REVIEW

**Review by aspain — APPROVED** [review]
Thanks @paolaperaza looks good to me

File: astro/create-project.md [hunk 1]
```diff
@@ -22,8 +22,20 @@ To create an Astro project, you need:
...
+```
+
+Then, navigate to that directory.
+
+```
+cd <your-astro-project>
+```
+
+Now, initialize an Astro project within that empty directory. To do so, run:
+
```
Comment (jwitz) [inline astro/create-project.md:38]:
```suggestion
To create a new Astro project, 

1. Run the following command to create a new directory for your Astro project:

```
mkdir <your-astro-project-name>
```

2. Open the directory: 
 
    ```
    cd <your-project-directory>
    ```

3. Run the following command to initialize an Astro project within the directory:
```

**Review by jwitz — APPROVED** [review]
GH suggestions were giving me trouble, so I jumped in and made mostly structural edits here.  Cut some language like "Desired name" which made it sound like the directory name will have some future role in the project/ configurations (it doesn't right? )

**Comment by aspain** [comment]
> GH suggestions were giving me trouble, so I jumped in and made mostly structural edits here. Cut some language like "Desired name" which made it sound like the directory name will have some future role in the project/ configurations (it doesn't right? )

@jwitz correct the directory name has no impact elsewhere

**Pull request merged**